SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -95,11 +97,22 @@ struct ConfigureConnection
 		Params.network.kcp.upstream_kcp.flush_interval_millis = Config.UdpUpstreamIntervalMS;
 		Params.network.kcp.downstream_kcp.flush_interval_millis = Config.UdpDownstreamIntervalMS;
 
+		const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
+
 #if WITH_EDITOR
-		Params.network.tcp.downstream_heartbeat = &HeartbeatParams;
-		Params.network.tcp.upstream_heartbeat = &HeartbeatParams;
-		Params.network.kcp.downstream_heartbeat = &HeartbeatParams;
-		Params.network.kcp.upstream_heartbeat = &HeartbeatParams;
+		Worker_HeartbeatParameters WithEditorHeartbeatParams = { Settings->HeartbeatIntervalSeconds * SECONDS_TO_MILLIS,
```

Comment:
Am I missing what is different between these `#if #else` blocks?

Reply:
I messed up. The second one should have "HeartbeatTimeoutSeconds" rather than "HeartbeatTimeoutWithEditorSeconds"